app/controllers/stripe_active_cards_controller.rb

```diff
@@ -1,13 +1,17 @@
 class StripeActiveCardsController < ApplicationController
   before_action :authenticate_user!
 
+  AUDIT_LOG_CATEGORY = "user.creditcard.modification".freeze
```

Comment:
I would call it `user.credit_card.edit`, mostly because `credit_card` are two separate words

Reply:
Sure, I was going back and forth on this, if you prefer the underscored version that settles it.